kernel/cpu_mask.c

```diff
@@ -66,11 +66,7 @@ int k_thread_cpu_mask_disable(k_tid_t thread, int cpu)
 
 int k_thread_cpu_pin(k_tid_t thread, int cpu)
 {
-	int ret;
+	uint32_t  mask = BIT(cpu);
```

Comment:
The CI warning here is legit

Reply:
@ycsin - The latest push should fix that CI warning.